### PR TITLE
xposting opensource.com article

### PR DESCRIPTION
**Pull request by Erioldoesdesign**
xposting the open source design article from Opensource.com

### CODE REVIEW

File: _posts/Use-your-artistic-skills-to-help-open-source [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Repost from OpenSource.com for an article from [Jeff Macharyas](https://opensource.com/users/jeffmacharyas) about Open Source Software projects and how Open Source Design connects projects that need design help with the designers who can help them.
```
Comment (RDIL) [inline _posts/Use-your-artistic-skills-to-help-open-source:1]: This needs the proper jekyll metadata.

Comment (Erioldoesdesign) [reply _posts/Use-your-artistic-skills-to-help-open-source:1]: Do you mean the part that looks like this:

---
layout: post
title: "Welcome to Open Source Design"
date: 2015-04-25
categories: design open source launching
author: bnvk
redirect_from:
  - /launching
permalink: /2015/04/25/welcome-to-open-source-design
---

Comment (RDIL) [reply _posts/Use-your-artistic-skills-to-help-open-source:1]: Yeah

**Comment by Erioldoesdesign** [comment]
Added the info needeed to post an article

**Comment by Erioldoesdesign** [comment]
@ei8fdb can you give this an approval for merging

**Review by RDIL — DISMISSED** [review]

**Pull request merged**